### ISSUE TITLE
cannot complete latex on win10

### ISSUE THREAD

**Issue by AStupidBear** [JuliaEditorSupport/julia-vscode#150]
![2017-04-07_184438](https://cloud.githubusercontent.com/assets/16422976/24797101/5b6bbbac-1bc2-11e7-81ef-27bb8737f203.jpg)
press tab then
![2017-04-07_184517](https://cloud.githubusercontent.com/assets/16422976/24797102/5b7890d4-1bc2-11e7-9c44-8c5615d1a043.jpg)

**Comment by kyungminlee** [comment]
+1. I have the same issue.

**Comment by ZacLN** [comment]
Hmm, what version are you each on?

**Comment by AStupidBear** [comment]
@ZacLN Do you mean the version of julia plugin? It's 0.5.1.

**Comment by ZacLN** [comment]
Yep I do, this is fixed on master and I think we'll be pushing a new release very soon

**Comment by cossio** [comment]
Same issue here. Linux Mint 18, vscode 1.12.2, julialang.language-julia 0.5.1

**Comment by cossio** [comment]
Note that in the animation here:

https://github.com/JuliaEditorSupport/julia-vscode/wiki/Snippets#latex

the user gets 3 completion option that we are not seeing. We only see `\eta` with an icon that looks like a blue box. But the animation shows two additional `\eta`s (one with capital E) with a different icon.

![image](https://cloud.githubusercontent.com/assets/10637482/26376145/87d8eca6-3fd9-11e7-854a-cfcd238684ed.png)

vs.

![image](https://cloud.githubusercontent.com/assets/10637482/26376176/9f5dc93c-3fd9-11e7-83a5-620009e1bb72.png)

**Comment by davidanthoff** [comment]
I'm closing this because we think this is fixed for v0.6.0. We will have a beta out for that version today, and hopefully a release next week. If you still see this problem on v0.6.0, please report back here and I'll reopen the issue.

**Comment by veritas9872** [comment]
@davidanthoff  Hello, I am having the same problem on Windows 10 Julia VSCode extension version 1.0.10 and Julia VSCode Insiders extension version 1.0.11.

**Comment by eranschweitzer** [comment]
@veritas9872 For what it's worth, I came across this thread because I thought I had the same issue (I'm also working with Windows 10). However, I then turned back to VSCode and suddenly something like `\alpha` + <kbd>TAB</kbd> was suddenly working.
I think in my case it's simply the extension being slow to load?
Maybe that is not the problem you were having, but it might be worth just waiting a minute and trying again.
